js/privatebin.js

```diff
@@ -2300,11 +2308,18 @@ jQuery.PrivateBin = (function($, RawDeflate) {
         {
             // show new bootstrap method (if available)
             if ($passwordModal.length !== 0) {
-                $passwordModal.modal({
-                    backdrop: 'static',
-                    keyboard: false
-                });
-                $passwordModal.modal('show');
+                if (typeof bootstrap !== 'undefined' && bootstrap.Tooltip.VERSION) {
+                    (new bootstrap.Modal($passwordModal[0]), {
```

Comment:
The right parenthesis `)` was misplaced. Should be `(new bootstrap.Modal($passwordModal[0], { ... })).show()`

Reply:
Doing that gives me an error. The single closing parenthesis captures the new object, on which the show() method is called.

Reply:
Sorry for the confusion. The Modal constructor accepts two parameters, but in the commented code, the closing parenthesis was placed right after the first parameter. Therefore, the config object was not passed in, and the `show()` method was called on the config object instead of the modal instance.

Reply:
Thank you, that does indeed work. Fixed!